stacks/api/function/enqueuer/src/firehose.ts

```diff
@@ -0,0 +1,142 @@
+import {EventQueue, FirehoseQueue} from "@spica-server/function/queue";
+import {Event, Firehose} from "@spica-server/function/queue/proto";
+import * as url from "url";
+import * as ws from "ws";
+import {Description, Enqueuer} from "./enqueuer";
+import express = require("express");
+
+interface FirehoseOptions {
+  event: "*" | "**" | "connection" | "close" | string;
```

Comment:
@thesayyn What's the meaning of these "*" and "**" event types?

Reply:
The wildcard '*' will match with everything. 
The wildcard '**' will only match with 'connection' and 'close' events.